programs/marinade-finance/src/instructions/admin/config_marinade.rs

```diff
@@ -88,7 +88,7 @@ impl<'info> ConfigMarinade<'info> {
         let min_stake_change = if let Some(min_stake) = min_stake {
             require_gte!(
                 min_stake,
-                5 * self.state.rent_exempt_for_token_acc,
+                State::MIN_STAKE_MULTIPLIER * self.state.rent_exempt_for_token_acc,
```

Comment:
Remove MIN_STAKE_MULTIPLIER. According to the new solana code the min_stake value must be at least 1 SOL, so please use LAMPORTS_PER_SOL or make another constant = it instead of multiplication